contracts/database/orm/factory.go

```diff
@@ -0,0 +1,12 @@
+package orm
+
+//go:generate mockery --name=Factory
+type Factory interface {
+	Count(count int) Factory
+	Raw() any
+	Create() error
+	CreateQuietly() error
```

Comment:
S: We can use it like below, we may remove the `Model` function if we implement it like this:

```
var user models.User
facades().Orm().Factory().Create(&user)

var user []models.User
facades().Orm().Factory().Times(3).CreateQuietly(&user)
```

```suggestion
	Create(any) error
	CreateQuietly(any) error
```